internal/delete_handler.go

```diff
@@ -328,18 +379,28 @@ func HandleDeleteRetain(folder storage.Folder, args []string, confirmed bool,
 	tracelog.ErrorLogger.FatalOnError(err)
 }
 
+func extractDeleteEverythingModifierFromArgs(args []string) int {
+	if len(args) == 0 {
+		return NoDeleteModifier
+	} else if args[0] == StringModifiersDeleteEverything[0] {
+		return IgnorePermanentDeleteModifier
+	} else {
```

Comment:
If someone makes a typo like `delete everything blabla` he invokes `delete everything EXCEPT_PERM`. That's not cool.

Reply:
``delete everything blabla`` doesn't work because of cobra
only "delete everything", "delete everything EXCEPT_PERM" or "delete everything INCLUDE_PERM"